src/main/java/pl/gov/coi/pomocua/ads/BaseOffer.java

```diff
@@ -24,8 +24,8 @@
     @GeneratedValue(strategy = IDENTITY)
     public Long id;
 
-    @NotNull
```

Comment:
Shouldn't it **be** `@NotNull`? We just don't want to return it back to the front-end.
We wouldn't allow guests (not logged in users) to create an offer. What do you think?
If so, then the `sampleOfferRequest()` methods in tests should be corrected as well.

Reply:
Nevermind, Hubert explained me why it have to be deleted from here (to prevent validation of that field on endpoint side)